templates/cassiopeia/index.php

```diff
@@ -142,7 +149,7 @@
     . $hasClass
     . ($this->direction == 'rtl' ? ' rtl' : '');
 ?>">
-    <header class="header container-header full-width<?php echo $stickyHeader ? ' ' . $stickyHeader : ''; ?>">
+    <header class="header container-header full-width<?php echo $stickyHeader ? ' ' . $stickyHeader : ''; echo ' ' . $navPadding;?>">
```

Comment:
```suggestion
    <header class="header container-header full-width<?php echo ($stickyHeader ? ' ' . $stickyHeader : '') . $navPadding; ?>">
```
PHPCS fails in Drone because there a multiple PHP statements in one line. (A statement is ended with a semicolon.)

In addition, your code produces an unnecessary space at the end of the class when `$navPadding` is an empty string.

This change suggestion should fix both (the latter in combination with my previous change suggestion).